### PR TITLE
Use posix `sh` for systems without `bash`

### PR DESCRIPTION
**Pull request by uberbaud**
This is a resubmission of my previous pull request because I don't (yet) know a better way to sync current/master.

### CODE REVIEW

File: src/libpostal_data [hunk 1]
```diff
@@ -79,9 +78,13 @@ function download_multipart() {
         else
             max=$size;
         fi;
-        printf "%s\0%s\0%s\0%s\0%s\0" "$i" "$offset" "$max" "$url" "$part_filename"
+        download_part "$i" "$offset" "$max" "$url" "$part_filename" &
+        # wait every time we have started $num_workers processes
+        [ $((i%num_workers)) -eq 0 ] && wait
```
Comment (uberbaud) [inline src/libpostal_data]: Use shell job control and modulo arithmetic to restrict maximum simultaneous downloads, could have also used `xargs -t` to show per download commands, or `xargs external_cmd`, or `xargs bash $STRING_OF_ALL_CMDS` just as easily. This was just closest to my original change.

**Comment by thatdatabaseguy** [comment]
I think I prefer Option 1 or 2 as mentioned in #90 rather than using a barrier at each block. With an ```xargs``` pool, worker processes pick up the next job in the queue as soon as they're available. Since the last chunk can be larger than the previous batch of normal-sized chunks, it's best to start on it as soon as possible.

**Comment by uberbaud** [comment]
I went with option 1. Secretly, I had chosen 3 because between options 1 & 2 I couldn't decide whether clarity of code or clarity of user messages was more important. I had hoped you would come down strongly for one or the other. Oh, well. ;)

File: src/libpostal_data [hunk 2]
```diff
@@ -117,7 +110,7 @@ function download_file() {
         content_length=$(curl -I $url 2> /dev/null | awk '/^Content-Length:/ { print $2 }' | tr -d '[[:space:]]')
 
         if [ $content_length -ge $LARGE_FILE_SIZE ]; then
-            download_multipart $url $local_path $content_length $NUM_WORKERS
+            download_multipart $url $local_path $content_length
```
Comment (uberbaud) [inline src/libpostal_data]: `download_multipart` has not been using the passed argument (which becomes `num_workers`) so don't pass it.

**Comment by thatdatabaseguy** [comment]
Looks good. Messages are preferable here since it's a heftier download than most people are probably used to.

**Pull request merged**